crates/anemo-cli/src/util.rs

```diff
@@ -11,7 +11,7 @@ async fn noop_handle(_request: Request<Bytes>) -> Result<Response<Bytes>, Infall
 pub async fn create_client_network(address: &str, server_name: &str) -> Result<(Network, Peer)> {
     let network = Network::bind("0.0.0.0:0")
         .private_key(random_key())
-        .server_name(server_name)
```

Comment:
I would vote to keep this called `server_name` instead of changing, since it wouldn't require existing clients to change code and we normally don't expect to see the secondary variant used.

So (just an opinion, but) my preference would be for the names to be simply `server_name` and `alternate_server_name` or something like that names to be simply `server_name` and `alternate_server_name` or something like that. (You could keep `secondary` if you want but `alternate` to me is slightly better in evoking the more temporary status)

Reply:
>  it wouldn't require existing clients to change code

is a great point, I will change it back.

Also will use `alternate` as you suggested